### DISCUSSION TITLE
Welcome to thrift-syntax-support Discussions!

### DISCUSSION THREAD

**Discussion by MrKou47** [MrKou47/thrift-syntax-support#8] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JooDye** [comment]
Hi, may I know what vscode theme is this? the one in the intro page. 
![image](https://github.com/MrKou47/thrift-syntax-support/assets/33674169/5e363353-5e93-47fe-a954-d3c9abaa23be)

**Reply by MrKou47** [reply]
It's the **Solarized Light** theme from the default preset of VSCode :)
